pkg/assembler/backends/neo4j/hasSLSA.go

```diff
@@ -488,23 +488,20 @@ func (c *neo4jClient) HasSlsa(ctx context.Context, hasSLSASpec *model.HasSLSASpe
 
 }
 
-// TODO (pxp928): combine with testing backend in shared utility
+// TODO(pxp928): combine with testing backend in shared utility
 func checkHasSLSAInputs(hasSLSASpec *model.HasSLSASpec) error {
-	invalidSubject := false
-	if hasSLSASpec.Package != nil && hasSLSASpec.Source != nil && hasSLSASpec.Artifact != nil {
-		invalidSubject = true
-	}
-	if hasSLSASpec.Package != nil && hasSLSASpec.Source != nil {
-		invalidSubject = true
+	subjectsDefined := 0
+	if hasSLSASpec.Subject.Package != nil {
```

Comment:
need a check here that `hasSLSASpec.Subject != nil` as that is a valid case

Reply:
This is now handled by where this gets called from